pkgs/applications/science/misc/openmodelica/omparser/default.nix

```diff
@@ -0,0 +1,27 @@
+{ pkg-config
+, jre8
+, libuuid
+, openmodelica
+, mkOpenModelicaDerivation
+}:
+
+mkOpenModelicaDerivation rec {
+  pname = "omparser";
+  omdir = "OMParser";
+  omdeps = [ openmodelica.omcompiler ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ jre8 libuuid ];
+
+  patches = [ ./Makefile.in.patch ];
+
+  meta = with lib; {
```

Comment:
You didn't import lib here

Reply:
What a shame. I even didn't check the buildability after made changes.

Thanks! Corrected.